app/data/subregion.json

```diff
@@ -64,7 +64,7 @@
+{"type":"Feature","geometry":{"type":"MultiPolygon","coordinates":[[[[-73.754687,40.610349],[-73.73799,40.594605],[-73.814331,40.583126],[-73.935316,40.545043],[-73.925427,40.561628],[-73.876246,40.569404],[-73.853016,40.581269],[-73.821436,40.586792],[-73.778579,40.609696],[-73.754687,40.610349]]],[[[-74.025186,40.705921],[-74.013784,40.756602],[-73.963183,40.8269],[-73.919881,40.913399],[-73.889377,41.024564],[-73.887315,41.127337],[-73.971017,41.247693],[-73.917966,41.307862],[-73.848575,41.23906],[-73.757973,41.344901],[-73.69569,41.294958],[-73.628441,41.300933],[-73.621494,41.255405],[-73.567535,41.259861],[-73.514249,41.198651],[-73.727776,41.100696],[-73.660895,40.987391],[-73.8172631,40.8138328],[-73.8963137,40.8067099],[-73.97827,40.710484],[-74.011236,40.700862],[-74.025186,40.705921]]],[[[-73.746803,40.78041],[-73.743952,40.637574],[-73.931445,40.575775],[-74.001958,40.571313],[-74.040367,40.61531],[-74.018293,40.674379],[-73.9962376,40.7033551],[-73.969273,40.70703],[-73.909626,40.790938],[-73.746803,40.78041]]]]},"properties":{"name":"NYCW","gridLoss":{"display":"4.88%","value":0.0488},"fuelMixCategories":{"renewable":0.9,"non-renewable":99.1,"non-renewable (excluding nuclear)":61.5,"renewable (excluding hydro)":0.9,"nuclear":37.6,"hydro":0},"fullName":"NPCC NYC/Westchester","emissionFactor":{"co2EmissionRate":{"value":596.4,"display":"596.4","units":"lb/MWh"},"noxEmissionRate":{"value":0.3,"display":"0.3","units":"lb/MWh"},"so2EmissionRate":{"value":0.02,"display":"0.02","units":"lb/MWh"}},"fuelMix":{"coal":0,"oil":1.4,"gas":60.1,"nuclear":37.6,"hydro":0,"biomass":0.9,"wind":0,"solar":0,"geothermal":0,"otherFossilFuel":0,"otherUnknownFuel":0}}},
```

Comment:
I'm tempted to fix the prep scripts before bringing in this change, but if the number should be updated beforehand, this is a great way to do so.

Reply:
Agreed. I reverted the respective commits and will close this PR.